src/palindrom.js

```diff
@@ -455,24 +495,28 @@ const Palindrom = (() => {
 
           if (res) {
             var statusCode = res.status;
-            var statusText = res.statusText;
+            var statusText = res.statusText || res.data;
             var reason = res.data;
           } else {
             // no sufficient error information, we need to create on our own
             var statusCode = -1;
-            var statusText = `An unknown network error has occurred. Raw message: ${error.message}`;
+            var statusText = `An unknown network error has occurred. Raw message: ${
+              error.message
+            }`;
             var reason = 'Maybe you lost connection with the server';
             // log it for verbosity
             console.error(error);
           }
+
+          const message = [
+            statusText,
+            ` statusCode: ${statusCode}`,            
+            ` reason: ${reason}`,
+            ` url: ${res.url}`
+          ].join('\n');
```

Comment:
Why not use just multiline string?

Reply:
Because the whitespace on the left would be included.